### PR TITLE
Updated Firecracker doc.

### PR DESCRIPTION
**Pull request by iulianbarbu**
Updated README.md Firecracker documentation

## Reason for This PR

Documentation should be updated to reflect current FC implementation.

## Description of Changes

README.md documentation was falling behind current Firecracker implementation. Now,
Firecracker supports AMD and ARM CPUs. Also, after building FC, the resulted binaries will
reside in a different directory than the one specified in the current documentation.
Changes: README.md, docs/getting-started.md

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria. Where there are two options, keep one.]`  
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] Either this PR is linked to an issue, or, the reason for this PR is
      clearly provided. 
- [x] The description of changes is clear and encompassing.
- [x] Either no docs need to be updated as part of this PR, or, the required
      doc changes are included in this PR. Docs in scope are all `*.md` files
      located either in the repository root, or in the `docs/` directory.
- [x] Either no code has been touched, or, code-level documentation for touched
      code is included in this PR.
- [x] Either no API changes are included in this PR, or, the API changes are
      reflected in `firecracker/swagger.yaml`.
- [x] Either the changes in this PR have no user impact, or, the changes in
      this PR have user impact and have been added to the `CHANGELOG.md` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,8 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
 has a minimalist design. It excludes unnecessary devices and guest-facing
 functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
-hardware utilization. Firecracker currently supports Intel CPUs, with planned
-AMD and Arm support. Firecracker will also be integrated with popular container
-runtimes.
+hardware utilization. Firecracker currently supports Intel, AMD and Arm CPUs. 
+Firecracker will also be integrated with popular container runtimes.
```
Comment (dianpopa) [inline README.md]: There are some container runtimes which are currently integrated with Firecracker. I d rephrase this sentence for accuracy reasons to something like: "Firecracker can also be integrated in container runtimes. Checkout [Kata Containers with Firecracker](https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support)".

File: README.md [hunk 2]
```diff
@@ -19,10 +19,10 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
 has a minimalist design. It excludes unnecessary devices and guest-facing
 functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
-hardware utilization. Firecracker currently supports Intel CPUs, with planned
-AMD and Arm support. Firecracker will also be integrated with popular container
-runtimes.
-
+hardware utilization. Firecracker currently supports Intel, AMD and Arm CPUs. 
+Firecracker can also be integrated in container runtimes. Checkout
```
Comment (acatangiu) [inline README.md]: nit:
```suggestion
Firecracker can also be integrated in container runtimes. Check out 
```

Comment (iulianbarbu) [reply README.md]: Fixed it here: https://github.com/firecracker-microvm/firecracker/compare/97edf46d827772c879f2d0a00c67df2597279cd4..909692079f3b6d3fc894f9e74ffbc0bf023403c2.

File: README.md [hunk 3]
```diff
@@ -19,10 +19,10 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
 has a minimalist design. It excludes unnecessary devices and guest-facing
 functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
-hardware utilization. Firecracker currently supports Intel CPUs, with planned
-AMD and Arm support. Firecracker will also be integrated with popular container
-runtimes.
-
+hardware utilization. Firecracker currently supports Intel, AMD and Arm CPUs.
```
Comment (sandreim) [inline README.md]:
```suggestion
hardware utilization. Firecracker currently supports Intel, AMD (beta) and ARM CPUs. 
```

Comment (dianpopa) [reply README.md]: ARM support is also considered beta.

Comment (acatangiu) [reply README.md]: @iulianbarbu @dianpopa fixd

Comment (acatangiu) [reply README.md]: @sandreim @dianpopa fixd

File: README.md [hunk 4]
```diff
@@ -19,10 +19,10 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
 has a minimalist design. It excludes unnecessary devices and guest-facing
 functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
-hardware utilization. Firecracker currently supports Intel CPUs, with planned
-AMD and Arm support. Firecracker will also be integrated with popular container
-runtimes.
-
+hardware utilization. Firecracker currently supports Intel, AMD (beta) and Arm
```
Comment (raduweiss) [inline README.md]: I propose using "public preview" or "preview" for the step between development and general availability. This is consistent with both our [roadmap](https://github.com/firecracker-microvm/firecracker/projects/13) and a more user-oriented view of things.

File: README.md [hunk 5]
```diff
@@ -19,10 +19,10 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
...
+(beta) CPUs. Firecracker can also be integrated in container runtimes. Checkout
+[Kata Containers with Firecracker.](https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support)
```
Comment (raduweiss) [inline README.md]: We should also mention [Ignite](https://github.com/weaveworks/ignite)

**Pull request merged**